content/docs/zero/install.mdx

```diff
@@ -93,6 +60,39 @@ docker compose up -d
 
 Pomerium requires an internet-accessible server and the ability to bind port 443.
 
+</TabItem>
+<TabItem value="linux" label="Linux">
+
+Install Pomerium with the following shell script:
+
+```bash
+curl https://console.pomerium.app/install.bash | env POMERIUM_ZERO_TOKEN=<CLUSTER_TOKEN> bash -s install
+```
+
+Pomerium requires an internet-accessible server and the ability to bind port 443.
+
+</TabItem>
+<TabItem value="kubernetes" label="Kubernetes">
+
+First, install Pomerium into your Kubernetes cluster:
+
+```bash
+kubectl apply -k github.com/pomerium/pomerium/k8s/zero\?ref=v0.26.0
```

Comment:
@wasaga does this need to be `ref=main` for now, to match https://github.com/pomerium/pomerium-zero/pull/2972?

Reply:
I'll be updating version tags throughout the docs as part of the docs release process. I've added this one to the list.